tgui/packages/tgui-panel/styles/tgchat/chat-dark.scss

```diff
@@ -405,6 +405,16 @@ em {
   font-weight: bold;
 }
 
+.dev_channel {
+  color: #775bff;
```

Comment:
These colors are really similar to deadchat colors from what I can see, maybe it should have a unique color. Maybe something green or pink

Reply:
They are the same as the mentor channel, might be worth a separate re-evaluation of our chat colours for these things